TypeScript.YAML-tmLanguage

```diff
@@ -23,14 +23,14 @@ repository:
 
   control-statement:
     name: keyword.control.ts
-    match: \b(break|catch|continue|debugger|declare|do|else|finally|for|if|return|switch|throw|try|while|with|super)\b
+    match: \b(break|catch|continue|debugger|declare|do|else|finally|for|if|return|switch|throw|try|while|with|super|case|default)\b
```

Comment:
I left a comment on the test-case also... but perhaps we should check keywords are not preceded by a "." . Reserved words are valid identifierNames, but not valid identifiers (i.e. "foo.return" is valid and "return" is not a keyword here, but "var return" is invalid as 'return' is a reserved work in this position).